core/vcpu.c

```diff
@@ -1991,8 +1995,11 @@ static int vcpu_emulate_insn(struct vcpu_t *vcpu)
     em_ctxt->rip = rip;
     rc = em_decode_insn(em_ctxt, instr);
     if (rc < 0) {
-        hax_panic_vcpu(vcpu, "%s: em_decode_insn() failed: vcpu_id=%u",
-                       __func__, vcpu->vcpu_id);
+        hax_panic_vcpu(vcpu, "em_decode_insn() failed: vcpu_id=%u,"
```

Comment:
Should we print these info as well?
ctxt->mode, ctxt->override_operand_size

Reply:
Ah, sorry I missed this comment. I'll instead add a call to `dump_vmcs()`, which will give us (among other info) guest CR0 (`GUEST_CR0` of VMCS) and CS (`GUEST_CS_AR` of VMCS), from which we'll be able to derive `ctxt->mode`, etc.